portfolioAstro/src/styles/melo.scss

```
@@ -0,0 +1,484 @@
+@keyframes fall-in-illust {
+  0% {
+    transform: translateY(-500px);
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+.select-menu {
+  @include flex-space-between;
+  margin-bottom: 50px;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)